roles/ci_gen_kustomize/README.md

```diff
@@ -0,0 +1,28 @@
+# ci_gen_kustomize
+Generate complete kustomization file based on multiple snippets
```

Comment:
Can we get a better name?

The CR that is generated isn't a kustomization file. It's a `ConfigMap` which kustomize can read. I've been calling it a kustomize data file or a kustomize values file. Maybe one of these names would work?

`ck_gen_kust_cf`
`ck_gen_kustomize_cf`
`ci_gen_kustomize_configmap`
`ci_gen_kustomize_values`